pkg/kube/krt/collection.go

```diff
@@ -463,21 +463,10 @@ func (h *manyCollection[I, O]) GetKey(k Key[O]) (res *O) {
 	return nil
 }
 
-func (h *manyCollection[I, O]) List(namespace string) (res []O) {
```

Comment:
so how can get by namespace now

Reply:
you can make an index and query that. note we didn't have any code currently listing by namespace anyways